include/boost/geometry/algorithms/detail/buffer/buffer_box.hpp

```diff
@@ -11,16 +11,19 @@
 
 #include <cstddef>
 
+#include <boost/geometry/core/access.hpp>
 #include <boost/geometry/core/coordinate_dimension.hpp>
 #include <boost/geometry/core/coordinate_type.hpp>
-#include <boost/geometry/core/access.hpp>
-
 
-namespace boost { namespace geometry
+namespace boost
+{
+namespace geometry
```

Comment:
I cannot find a rule which doesn't affect this style for namespace.
Maybe we can live with this change.

Reply:
With the more detailled Brace rules as in https://github.com/boostorg/geometry/pull/1400#issuecomment-2833213438 , I can get `namespace boost { namespace geometry {`

Reply:
Yes, better for sure! Thanks! Applied!